### ISSUE TITLE
comparing rasch.jml to winsteps part 2

### ISSUE THREAD

**Issue by thoughtafter** [alexanderrobitzsch/sirt#5]
We have tried using rasch.jml with constraints to fix some items. We are finding that compared to Winsteps the item difficulties are off by +/- .02 logits. We've also tried tam.jml, which has slightly different numbers but has a similar margin of error.

Settings:
````
sirt::rasch.jml(dat, method="WLE", wle.adj=0.3, constraints=anchor)
tam.jml(tamdat, bias=FALSE, xsi.fixed=anchor)
````
We don't need the numbers to match Winsteps exactly but they need to be closer than they are now. If you can shed any insight on this it would be greatly appreciated.

**Comment by alexanderrobitzsch** [comment]
Could you please send me the dataset and the Winsteps output you want to 
have. Then I can figure out the estimation settings. You can alo use 
some dataset in sirt and replicate the analysis in Winsteps and send me 
the output.

**Comment by thoughtafter** [comment]
I have emailed you the information you requested.

**Comment by alexanderrobitzsch** [comment]
I tried several argument value combinations and it seems that `sirt::rasch.jml()` mimics [WinSteps](http://www.winsteps.com/index.htm) with the argument `method="MLE"` (leading to a maximum absolute difference of .003). This means that WinSteps uses maximum likelihood estimates (MLE) as person parameter estimates during estimation. By default, `TAM::tam.jml()` and `sirt::rasch.jml()` use weighted likelihood estimates (WLE).
